pyproject.toml

```diff
@@ -4,7 +4,7 @@ version = "0.4.4-dev"
 description = "A scanner that will scan your AI models for problems. Currently it focuses on bias testing. It is currently alpha."
 authors = ["Your Name <you@example.com>"]
 license = "Apache-2.0"
-readme = "README.rst"
+# readme = "./daisybell/README.rst"
```

Comment:
Instead of removing the reference to the README in the packaging file, please copy `README.rst` into the container. This file is documentation that users may access via python/pip commands and should be present